src/plugin-slots/LogoSlot/README.md

```diff
@@ -0,0 +1,69 @@
+# Logo Slot
+
+### Slot ID: `logo_slot`
+
+## Description
+
+This slot is used to replace/modify/hide the logo.
+
+## Examples
+
+### Modify URL
+
+The following `env.config.jsx` will modify the link href for the logo.
+
+```jsx
+import { PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+
+const modifyLogoHref = ( widget ) => {
+  widget.content.href = "https://openedx.org/";
+  return widget;
+};
+
+const config = {
+  pluginSlots: {
+    logo_slot: {
+      keepDefault: true,
+      plugins: [
+        {
+          op: PLUGIN_OPERATIONS.Modify,
+          widgetId: 'default_contents',
+          fn: modifyLogoHref,
+        },
+      ]
+    },
+  },
+}
+
+export default config;
+```
+
+### Custom Component
+
+The following `env.config.jsx` will replace the logo entirely (in this case with a centered 🗺️ `h1`)
+
+```jsx
+import { DIRECT_PLUGIN, PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+
+const config = {
+  pluginSlots: {
+    logo_slot: {
+      keepDefault: false,
+      plugins: [
```

Comment:
Just as an aside while we're all talking about the syntax for plugins, I have to say... that's a whole lot of code to replace a logo. 😬

Reply:
Yeah, I think people generally agree that FPF config feels a bit too verbose (hence the thread https://discuss.openedx.org/t/proposal-simplified-frontend-plugin-api-config/13312)

That being said, just changing the logo image itself is still very much possible via existing config methods - wrapping the Logo component in a PluginSlot doesn't change that